src/camera_system.py

```diff
@@ -31,3 +32,9 @@ def start(self) -> None:
                 print(state)
             # 2秒待つ
             time.sleep(2)
+
+        tt = TrainTracker(0)
+        # Webカメラのキャリブレーション
+        tt.calibrate()
+        # IoT列車の監視を開始
+        tt.observe()
```

Comment:
```suggestion
        tt.observe()
        
        # 列車攻略が終わったらインスタンスを破棄する
        del tt
```
あんまりインスタンスを破棄したい理由がわからんのだが、するとしたら列車攻略が終わったら？